### PR TITLE
update images update ubuntu centos-stream-9

### PR DESCRIPTION
**Pull request by why168**
1. [update ubuntu 24.04 release-20240710](https://github.com/lima-vm/lima/commit/1ce07ab62dabb830fc304e9b32c06591e9bc5171)
2. [update centos-stream-9 20240715](https://github.com/lima-vm/lima/commit/1b30788e11ca346a631fbee28f3523be7586c197)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@ _output/
 _artifacts/
 lima.REJECTED.yaml
 .config
+.idea/
+.DS_Store
```
Comment (jandubois) [inline .gitignore:6]: These patterns are not specific to the Lima project and therefore don't belong into the repo.

>Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.

(from https://git-scm.com/docs/gitignore)

File: examples/experimental/riscv64.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 arch: "riscv64"
 images:
-- location: "https://cloud-images.ubuntu.com/releases/24.04/release-20240423/ubuntu-24.04-server-cloudimg-riscv64.img"
+- location: "https://cloud-images.ubuntu.com/releases/24.04/release-20240710/ubuntu-24.04-server-cloudimg-riscv64.img"
```
Comment (AkihiroSuda) [inline examples/experimental/riscv64.yaml:5]: digest doesn't seem updated

File: examples/experimental/armv7l.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@
 arch: "armv7l"
 images:
 # Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
-- location: "https://cloud-images.ubuntu.com/releases/24.04/release-20240423/ubuntu-24.04-server-cloudimg-armhf.img"
+- location: "https://cloud-images.ubuntu.com/releases/24.04/release-20240710/ubuntu-24.04-server-cloudimg-armhf.img"
```
Comment (AkihiroSuda) [inline examples/experimental/armv7l.yaml:6]: digest doesn't seem updated

**Comment by AkihiroSuda** [comment]
Please squash commits

**Comment by AkihiroSuda** [comment]
Also needs sign-off

File: examples/apptainer-rootful.yaml [hunk 4]
```diff
@@ -4,12 +4,12 @@
 
 images:
 # Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
-- location: "https://cloud-images.ubuntu.com/releases/24.04/release-20240423/ubuntu-24.04-server-cloudimg-amd64.img"
+- location: "https://cloud-images.ubuntu.com/releases/24.04/release-20240710/ubuntu-24.04-server-cloudimg-amd64.img"
```
Comment (AkihiroSuda) [inline examples/apptainer-rootful.yaml:7]: Now https://cloud-images.ubuntu.com/releases/24.04/release-20240725/ is available

**Comment by why168** [comment]
digest sha256：https://cloud-images.ubuntu.com/releases/24.04/release-20240725/

File: examples/docker.yaml [hunk 5]
```diff
@@ -9,12 +9,13 @@
 # This template requires Lima v0.8.0 or later
 images:
 # Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
-- location: "https://cloud-images.ubuntu.com/releases/24.04/release-20240423/ubuntu-24.04-server-cloudimg-amd64.img"
+# ⚠️ release-20220309 is known to be broken on aarch64: https://github.com/lima-vm/lima/issues/712
```
Comment (AkihiroSuda) [inline examples/docker.yaml:12]: Why mention such ab old version?

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]
Please sign off the commits and squash them
(1 commit = 1 distro)

**Comment by AkihiroSuda** [comment]
See https://lima-vm.io/docs/community/contributing/ for how to sign off

**Comment by why168** [comment]
> Please sign off the commits and squash them (1 commit = 1 distro)



> 请签署提交并压缩它们 （1 个提交 = 1 个发行版）

finish

**Comment by AkihiroSuda** [comment]
Still contains an irrelevant merge commit

**Comment by why168** [comment]
> Still contains an irrelevant merge commit

fix

**Comment by AkihiroSuda** [comment]
DCO check is still failing
https://github.com/lima-vm/lima/pull/2495/checks

**Comment by AkihiroSuda** [comment]
https://github.com/lima-vm/lima/commit/7a23254de278b384d368ecc199087efb0af3bda6.patch

"Edwin.Wu" should be spelled as "Edwin Wu"

**Comment by AkihiroSuda** [comment]
> archlinux.yaml

Maybe this should be revisited in a separate PR

**Comment by why168** [comment]
> > archlinux.yaml
> 
> Maybe this should be revisited in a separate PR

yes， `archlinux.yaml` It's special. build is working fine

**Comment by why168** [comment]
> https://github.com/lima-vm/lima/commit/7a23254de278b384d368ecc199087efb0af3bda6.patch
> 
> "Edwin.Wu" should be spelled as "Edwin Wu"

fix

File: examples/almalinux-8.yaml [hunk 6]
```diff
@@ -4,9 +4,9 @@
 # EL9-based distros are known to work.
 
 images:
-- location: "https://repo.almalinux.org/almalinux/8.9/cloud/x86_64/images/AlmaLinux-8-GenericCloud-8.9-20231128.x86_64.qcow2"
+- location: "https://repo.almalinux.org/almalinux/8.10/cloud/x86_64/images/AlmaLinux-8-GenericCloud-8.10-20240530.x86_64.qcow2"
   arch: "x86_64"
-  digest: "sha256:a1686bc537bce699b512e3233666f5b8f69ed797ff1ce0af52c17fdc52942621"
+  digest: "sha256:41a6bcdefb35afbd2819f0e6c68005cd5e9a346adf2dc093b1116a2b7c647d86"
 - location: "https://repo.almalinux.org/almalinux/8.9/cloud/aarch64/images/AlmaLinux-8-GenericCloud-8.9-20231128.aarch64.qcow2"
```
Comment (AkihiroSuda) [inline examples/almalinux-8.yaml:10]: this should be updated too

**Comment by AkihiroSuda** [comment]
The commit message still inaccurately mention Arch Linux

File: examples/almalinux-8.yaml (hunk 6 above)
Comment (why168) [reply examples/almalinux-8.yaml]: finish

**Comment by AkihiroSuda** [comment]
ping @why168 
https://github.com/lima-vm/lima/pull/2495#issuecomment-2261005469

**Comment by why168** [comment]
> 提交消息仍然错误地提到了 Arch Linux

I can't fix it, someone is updating the Arch Linux  https://github.com/lima-vm/lima/pull/2524

**Comment by AkihiroSuda** [comment]
I don't understand.

**Comment by AkihiroSuda** [comment]
Images are outdated again.
Do you want to update?

**Comment by why168** [comment]
Next version update，🌈 v0.24.0

**Pull request merged**